keras_hub/src/models/retinanet/retinanet_image_converter.py

```diff
@@ -0,0 +1,52 @@
+from keras_hub.src.api_export import keras_hub_export
+from keras_hub.src.layers.preprocessing.image_converter import ImageConverter
+from keras_hub.src.models.retinanet.retinanet_backbone import RetinaNetBackbone
+from keras_hub.src.utils.tensor_utils import preprocessing_function
+
+
+@keras_hub_export("keras_hub.layers.RetinaNetImageConverter")
+class RetinaNetImageConverter(ImageConverter):
+    backbone_cls = RetinaNetBackbone
```

Comment:
The converter has no resizing option, this is something we need to support for vision models. The backbone is taking image_shape as an input. This means we have to support image resizing in preprocessor.